packages/server/test/mapping.test.ts

```diff
@@ -0,0 +1,90 @@
+import { uuid } from '@botpress/messaging-base'
```

Comment:
PP: IMHO it's better when tests take the same structure (or so) as the `src` folder